styleguide/source/_patterns/02-organisms/topic/topic-rail-right/topic-rail-right.twig

```diff
@@ -0,0 +1,3 @@
+<aside class="topic_rail-right">
```

Comment:
I'm not sure it makes sense to have separate organisms for the columns/rails. The rails can have variable content. I think it would be simpler/better to include the section organisms directly in the Template pattern.